onnx/checker.cc

```diff
@@ -228,6 +228,92 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& ctx) {
 #undef check_field
 }
 
+void check_sequence(const SequenceProto& sequence, const CheckerContext& ctx) {
+  enforce_has_field(sequence, elem_type);
+  if (sequence.elem_type() == SequenceProto::TENSOR) {
+    enforce_has_field(sequence, tensor_values);
```

Comment:
I think we can drop this check above, since tensor_values is a repeated field and since we (presumably) want to allow for the case with zero elements in the sequence.